R/compat-dplyr.R

```diff
@@ -14,13 +14,13 @@ dribble_is_reconstructable <- function(data) {
   is_list(data) &&
     has_dribble_cols(data) &&
     has_dribble_coltypes(data) &&
+    id_can_be_drive_id(data$id) &&
```

Comment:
When deciding if a `dribble` can be reconstructed, now we need to also consider if the putative `id` column can be cast to `drive_id`.